### DISCUSSION TITLE
Issue: Tenant - Missing required parameter for

### DISCUSSION THREAD

**Discussion by melvinmihaylovnulabg** [CharrafiMed/global-search-modal#69] [Q&A]
Hey, i think i have a problem with the Multi-tenancy, or it just might be a problem from me i am not sure.
I am getting an error that i can't really debug, the url is from my app and everything looks fine but i think it might be a problem with the global search. Correct me if i'am wrong. When i remove the plugin, everything works fine.

Missing required parameter for [Route: filament.crm.resources.crm-tasks.edit] [URI: crm/{tenant}/crm-tasks/{record}/edit] [Missing parameter: tenant].

Steps to reproduce: 
1. Login in the admin panel.
2. Delete session from database or redis or where you save user's session.
3. Close the web browser tab
4. Open new one and try to login

![image](https://github.com/user-attachments/assets/c0f39420-bcf3-4a55-8577-91645a5e8914)
![image](https://github.com/user-attachments/assets/676ca2aa-cfe0-4b55-87fb-714e8c00c27b)

**Comment by melvinmihaylovnulabg** [comment]
@CharrafiMed After some debugging it is indeed a problem with the global search. I hope that it is an easy fix.

**Comment by CharrafiMed** [comment]
hey @melvinmihaylovnulabg !
I am already with this issue, I spent a lot of time resolving it, and it seems to occur for a lot of other plugins as well and until now we didn't find any solution for that, however, this issue appears only outside auth system #57  and see https://filamentphp.com/plugins/charrafimed-global-search-modal#troubleshooting

**Reply by melvinmihaylovnulabg** [reply]
@CharrafiMed Thanks for the respond. I will try to use the plugin with the scopes and i hope that it will work for now, but i am not sure, because we are using spa functionality. I hope that v4 will be out soon and the plugin will be fixed, because i really like it.

**Reply by CharrafiMed** [reply]
hey @melvinmihaylovnulabg this has been fixed due the latest releases
